posthog/test/base.py

```diff
@@ -763,3 +763,24 @@ def _create_insight(
     insight = Insight.objects.create(team=team, filters=insight_filters)
     dashboard_tile = DashboardTile.objects.create(dashboard=dashboard, insight=insight)
     return insight, dashboard, dashboard_tile
+
+
+def create_person_id_override_by_distinct_id(distinct_id_from: str, distinct_id_to: str, team_id: int):
```

Comment:
reasonably rough for now but does the job nicely. as I go along I might set up some decorators and nicer helpers for testing